service/history/handler/handler.go

```diff
@@ -2069,7 +2068,7 @@ func (h *handlerImpl) RatelimitUpdate(
 func (h *handlerImpl) convertError(err error) error {
 	switch err := err.(type) {
 	case *persistence.ShardOwnershipLostError:
-		info, err2 := h.GetMembershipResolver().Lookup(service.History, strconv.Itoa(err.ShardID))
+		info, err2 := h.GetMembershipResolver().Lookup(service.History, string(rune(err.ShardID)))
```

Comment:
maybe as a followup PR, but could we have a function that accepts shardID and then hides this logic in 1 place? :)

Reply:
That's a good idea!
I'm afraid codecov for new lines won't pass this though :-(
Let me check what's with the coverage of the other multiple lines where `Lookup()` is used.

Reply:
Turns out, not that bad! Had to introduce a new directory to avoid cycle dependencies on `go-generate` though.